helpers.js

```diff
@@ -42,9 +33,37 @@ function logToActivityLog(message) {
   console.error(message);
 }
 
+async function validatePaths(paths) {
```

Comment:
Instead of telling what this function is doing (validating paths) tell what it achieves (asserts that paths exist).

My personal opinion on assert vs check is:
-Failing asserts results with an error
-Check is more abstract, and when concrete uses form `isValid...` and returns corresponding boolean result, not maent to throw errors.